requirements_dev.txt

```diff
@@ -0,0 +1,18 @@
+# CPU Compatible Requirements
+-r requirements.txt
+
+# Dev:
+# Development, documentation, testing related dependencies.
+pytest
+pytest-cov
+pytest-console-scripts
+pytest-ordering
```

Comment:
```suggestion
pytest-ordering~=0.6
```

Reply:
Yup. Will add this in as well. I was REALLY hoping that we will get CI to a state where daily CI runs will catch any subversion induced problems but I think there is a whole bunch of tests to enhance before that happens.